pkg/config/config.go

```diff
@@ -46,10 +47,17 @@ type Organization struct {
 	Policies map[string]Policy
 	MSP      MSP
 
-	AnchorPeers      []AnchorPeer
+	// AnchorPeers contains the endpoints of anchor peers for each application organization.
+	AnchorPeers      []Address
 	OrdererEndpoints []string
 }
 
+// Address contains the hostname and port for an endpoint.
```

Comment:
field is host, "host name" is English, and "hostname" is jargon. Which do we want?